src/ia-menu-slider.js

```diff
@@ -59,14 +71,31 @@ export class IAMenuSlider extends LitElement {
     return this.selectedMenu ? 'open' : 'closed';
   }
 
+  get closeAction() {
+    const menuItem = this.menus.find(menu => menu.id === this.selectedMenu);
+    if (!menuItem) { return nothing; }
```

Comment:
Interesting.. I've never used `nothing`. What is it used for here?

Reply:
https://lit-html.polymer-project.org/guide/writing-templates#rendering-nothing